package.json

```diff
@@ -74,5 +78,16 @@
       "pre-commit": "npm test",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "release": {
+    "plugins": [
```

Comment:
@nellyk, do we not need to install all these as devDependencies?

Reply:
for this plug in you don't have to https://semantic-release.gitbook.io/semantic-release/usage/plugins

![image](https://user-images.githubusercontent.com/3062772/75382242-06484f00-58d2-11ea-91f7-53ea13c39421.png)

Reply:
Then `@semantic-release/commit-analyzer` should be removed from the dependencies right? Or do we require it separately for something else?

Reply:
ooh yes that should be removed